pkg/arvo/app/landscape/js/channel.js

```diff
@@ -210,6 +212,9 @@ class Channel {
         if (obj.hasOwnProperty("err")) {
```

Comment:
There's a comment just above here (that gh won't let me select) that says "only notify on err". So that can be removed.

Reply:
fixed.